src/GitVersionCore.Tests/BuildServers/TeamCityTests.cs

```diff
@@ -19,7 +19,7 @@ public void SetUp()
             });
             buildServer = sp.GetService<TeamCity>();
         }
-        
```

Comment:
These are the gratitious whitespaces that are lying around in the repo waiting for an IDE/Editor to come and cleanup whitespace that needs trimming.